oak-core/src/main/java/org/apache/jackrabbit/oak/Oak.java

```diff
@@ -584,6 +584,10 @@ public Oak with(@NotNull Whiteboard whiteboard) {
             LOG.info("Registered Prefetch feature: " + QueryEngineSettings.FT_NAME_PREFETCH_FOR_QUERIES);
             closer.register(prefetchFeature);
             queryEngineSettings.setPrefetchFeature(prefetchFeature);
+            Feature improvedIsNullCostFeature = newFeature(QueryEngineSettings.FT_NAME_IMPROVED_IS_NULL_COST, whiteboard);
+            LOG.info("Registered improved cost feature: " + QueryEngineSettings.FT_NAME_IMPROVED_IS_NULL_COST);
```

Comment:
To be consistent, would be better to use string interpolation for all log statements.

Reply:
Actually it would make it inconsistent with the above line, which is:

      LOG.info("Registered Prefetch feature: " + QueryEngineSettings.FT_NAME_PREFETCH_FOR_QUERIES);